requirements.txt

```diff
@@ -21,4 +21,6 @@ streamlit<=1.28.0
 tqdm<=4.66.1
 unidecode<=1.3.7
 urllib3<=2.0.7
-XlsxWriter<=3.1.9
\ No newline at end of file
+XlsxWriter<=3.1.9
+Pyomo<=6.6.2
+Pillow<=10.0.1
```

Comment:
why is Pillow added to requirements? I do not see it used anywhere.

Reply:
It is the dependency of a dependency so it is no problem. And the requirements file is only used internally not for the building of the library.